### ISSUE TITLE
Dracut support?

### ISSUE THREAD

**Issue by deimosian** [picodotdev/alis#222]
Hey, just came across this and it looks much cleaner than my personal install scripts...

My only issue is that it does not appear to support using `dracut` instead of `mkinitcpio` for generation of initramfs or unified kernel images and that's a requirement for some of my systems and I use it everywhere now.

I may try and add it if I get time and a pull request for it would be welcomed.

**Comment by picodotdev** [comment]
Yes, please, send a pull request and I will review and merge it.
